src/esp/bindings/SimBindings.cpp

```diff
@@ -285,6 +285,11 @@ void initSimBindings(py::module& m) {
           "recompute_navmesh", &Simulator::recomputeNavMesh, "pathfinder"_a,
           "navmesh_settings"_a, "include_static_objects"_a = false,
           R"(Recompute the NavMesh for a given PathFinder instance using configured NavMeshSettings. Optionally include all MotionType::STATIC objects in the navigability constraints.)")
+      .def(
+          "convex_hull_decomposition", &Simulator::convexHullDecomposition,
```

Comment:
Nit: On the Simulator class bindings too, methods should be verbs,  so it should be apply_convex_hull_decomposition or convex_hell_composite? (Should definitely have convex hull decomposition somewhere though for the search term.

Reply:
Actually, Trimesh does this too, which is disappointing so I suppose it's fine.

Reply:
Still a good thought, I went ahead and changed it :)